src/foremast/awslambda/awslambda.py

```diff
@@ -211,7 +211,7 @@ def update_function_configuration(self, vpc_config):
 
         LOG.info("Successfully updated Lambda configuration.")
 
-    @retries(max_attempts=3, wait=1, exceptions=(SystemExit))
+    @retries(max_attempts=10, wait=3, exceptions=(SystemExit))
```

Comment:
I think the max_attempts is off, but the wait should be increased to `3` as you have done. 1 second delay is potentially too aggressive.

Reply:
I have replaced this with the backoff library per my initial recommendation in my pull request. I have changed the max attempts back to the original values per request